jdi-light-material-ui-tests/src/main/java/io/github/com/pages/inputs/RadioButtonPage.java

```diff
@@ -0,0 +1,36 @@
+package io.github.com.pages.inputs;
+
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+
+import java.util.List;
+
+public class RadioButtonPage extends WebPage {
+
+    @UI("//fieldset[@id='simpleRadio']//label")
```

Comment:
need to use locator
//fieldset[@id='simpleRadio']//span[contains(@Class,'MuiRadio-root')]

Reply:
label is needed for usage in simpleRadioButton.has().text()

Reply:
done